module/VuFind/src/VuFind/View/Helper/Root/Record.php

```diff
@@ -419,10 +419,10 @@ public function getSearchResult($view)
      */
     public function getCheckbox($idPrefix = '', $formAttr = false, $number = null)
     {
-        $id = $this->driver->getSourceIdentifier() . '|'
-            . $this->driver->getUniqueId();
-        $context
-            = ['id' => $id, 'number' => $number, 'prefix' => $idPrefix];
+        $context = compact('number') + [
+            'id' => $this->getUniqueIdWithSourcePrefix(),
+            'checkboxElementId' => $this->getUniqueHtmlElementId($idPrefix)
+            ];
```

Comment:
Minor style point: I'd de-indent the closing `];` here for consistency with code style elsewhere.